crates/forge/tests/integration/setup_fork.rs

```diff
@@ -0,0 +1,114 @@
+use crate::integration::common::corelib::{corelib_path, predeployed_contracts};
+use crate::integration::common::running_tests::run_test_case;
+use crate::{assert_passed, test_case};
+use camino::Utf8PathBuf;
+use forge::scarb::{ForgeConfig, PredefinedFork};
+use forge::{run, RunnerConfig};
+use indoc::formatdoc;
+use std::collections::HashMap;
+
+#[test]
+fn fork_simple_decorator() {
+    let test = test_case!(formatdoc!(
+        r#"
+            use result::ResultTrait;
+            use array::ArrayTrait;
+            use option::OptionTrait;
+            use traits::TryInto;
+            use starknet::ContractAddress;
+            use starknet::Felt252TryIntoContractAddress;
+            use starknet::contract_address_const;
+            use snforge_std::{{ BlockTag, BlockId }};
+
+            #[starknet::interface]
+            trait IHelloStarknet<TContractState> {{
+                fn increase_balance(ref self: TContractState, amount: felt252);
+                fn get_balance(self: @TContractState) -> felt252;
+            }}
+
+            #[test]
+            #[fork(url: "{}", block_id: BlockId::Tag(BlockTag::Latest))]
+            fn test_fork_simple() {{
+                let dispatcher = IHelloStarknetDispatcher {{
+                    contract_address: contract_address_const::<3216637956526895219277698311134811322769343974163380838558193911733621219342>()
+                }};
+
+                let balance = dispatcher.get_balance();
+                assert(balance == 2, 'Balance should be 2');
+
+                dispatcher.increase_balance(100);
+
+                let balance = dispatcher.get_balance();
+                assert(balance == 102, 'Balance should be 102');
+            }}
+        "#,
+        "http://188.34.188.184:9545/rpc/v0.4"
```

Comment:
What is this network? Shoudn't be stored in some global variable?

Reply:
Does this requires that I am connected to the interenet to run test?

Reply:
Yes, it requires that you are connected to internet to run the test. It is an address of our integration node. We do it the same way in cheatnet tests currently.